backend/src/main/resources/application-azure-demo.yaml

```diff
@@ -12,3 +12,5 @@ simple-report:
       - https://demo.simplereport.gov
 twilio:
   enabled: false
+features:
+  testCardRefactorEnabled: false
```

Comment:
Do we need this snippet for the other envs?
Or a default set in the base application yaml?

Reply:
A default was set in the separate feature flag PR #6652 that had it enabled except for being disabled on prod. These changes also disable it on the other higher level environments except for `training` which design is planning to use for user testing sessions with the new card